### PR TITLE
Tip 4 Tweaks

### PR DESCRIPTION
**Pull request by rasbt**
**Did you add yourself as a [contributor](https://github.com/Benjamin-Lee/deep-rules/blob/master/contributors.md) if this is your first contribution?**

- [x] Yes, I added myself or am already a contributor

I added a few words about data sharing to tip 4. Could be out of scope though. Let me know.

### CODE REVIEW

File: content/06.know-your-problem.md [hunk 1]
```diff
@@ -7,10 +7,11 @@ If so, obtaining this data (and associated metadata), and reviewing the study pr
...
+
 Information about the resources used, download dates, and dataset versions are critical to preserve.
-Doing so will help to minimize operational confusion and will increase the reproducibility of the analysis.
+Doing so will help to minimize operational confusion and will increase the reproducibility of the analysis. Best practices for reproducibility also include sharing your collected dataset and metadata along upon publication of your study, ideally in a public dataset repository if there are no ethical or privacy concerns and no copyright restrictions. While recommended and recognized dataset repositories may differ across disciplines, a list of general dataset repositories includes the Dryad repository [@doi:10.1038/npre.2010.4595.1] (https://datadryad.org/), Figshare [@doi:10.4103/0976-500X.81919] (https://figshare.com), Zenodo [@doi:10.3897/biss.3.37080] (https://zenodo.org), and the Open Science Framework [@doi:10.5195/jmla.2017.88] (https://osf.io). In addition, Gundersen et al.'s provide useful checklists summarizing best data sharing practices for reproducible research and open science.
```
Comment (Benjamin-Lee) [inline content/06.know-your-problem.md:14]:
```suggestion
Doing so will help to minimize operational confusion and will increase the reproducibility of the analysis. Best practices for reproducibility also include sharing your collected dataset and metadata along upon publication of your study, ideally in a public dataset repository if there are no ethical or privacy concerns and no copyright restrictions.
While recommended and recognized dataset repositories may differ across disciplines, a list of general dataset repositories includes the Dryad repository [@doi:10.1038/npre.2010.4595.1] (https://datadryad.org/), Figshare [@doi:10.4103/0976-500X.81919] (https://figshare.com), Zenodo [@doi:10.3897/biss.3.37080] (https://zenodo.org), and the Open Science Framework [@doi:10.5195/jmla.2017.88] (https://osf.io).
In addition, Gundersen et al.'s provide useful checklists summarizing best data sharing practices for reproducible research and open science.
```

Is there a citation for Gundersen et al?

File: content/06.know-your-problem.md [hunk 2]
```diff
@@ -7,10 +7,13 @@ If so, obtaining this data (and associated metadata), and reviewing the study pr
...
-Any of these data issues might limit a researcher's ability to use DL to address the biological question at hand, or might otherwise require adjustment before DL can be used.
+Any of these data issues might limit a researcher's ability to use DL to address the biological question at hand or might otherwise require adjustment before DL can be used.
 Data collection should also be carefully documented, or a data collection protocol should be created and specified in the project documentation.
+
 Information about the resources used, download dates, and dataset versions are critical to preserve.
-Doing so will help to minimize operational confusion and will increase the reproducibility of the analysis.
+Doing so will help to minimize operational confusion and will increase the reproducibility of the analysis. Best practices for reproducibility also include sharing your collected dataset and metadata along upon publication of your study, ideally in a public dataset repository if there are no ethical or privacy concerns and no copyright restrictions.
```
Comment (cgreene) [inline content/06.know-your-problem.md:14]:
```suggestion
Doing so will help to minimize operational confusion and will increase the reproducibility of the analysis.
Best practices for reproducibility also include sharing the collected dataset and metadata along upon publication of the study in a public dataset repository if there are no ethical or privacy concerns and no copyright restrictions.
```

**Review by cgreene — APPROVED** [review]
a couple suggested changes - one sentence per line and avoiding second person (which i feel like was discussed somewhere - but we'll need to do a finalr ead through for consistency)

File: content/06.know-your-problem.md (hunk 2 above)
Comment (rasbt) [reply content/06.know-your-problem.md:14]: Thanks! I was/am in favor of avoiding 2nd person but wasn't sure if we adopted it. I believe I saw it in several places when editing, so we probably need to double check regarding this again.

**Comment by rasbt** [comment]
Issues with 2nd person and multiple sentences per line should be addressed now.

**Comment by Benjamin-Lee** [comment]
Thanks for the fixes! I have been ignoring the second person changes in favor of a single read through focused on it in the future.

**Review by Benjamin-Lee — APPROVED** [review]

**Pull request merged**